### PR TITLE
added wordcloud

### PR DESCRIPTION
**Pull request by MahinRahman8901**

### CODE REVIEW

**Comment by MahinRahman8901** [comment]
closes #135

File: dashboard/streamlit_app.py [hunk 1]
```diff
@@ -189,6 +190,33 @@ def get_data_from_db(conn: connect) -> pd.DataFrame:
     return df
 
 
+def generate_word_cloud(summary_texts):
+    combined_text = ' '.join(summary_texts)
+    word_cloud = WordCloud(width=800, height=400, background_color='white',
```
Comment (ErvinRex) [inline dashboard/streamlit_app.py:195]: any particular reason for these sizes?

File: dashboard/streamlit_app.py [hunk 2]
```diff
@@ -189,6 +190,33 @@ def get_data_from_db(conn: connect) -> pd.DataFrame:
...
+                           stopwords=STOPWORDS).generate(combined_text)
+    return word_cloud
+
+
+def get_summary_texts_from_db(conn, case_no):
+    summary_texts = []
+    try:
```
Comment (ErvinRex) [inline dashboard/streamlit_app.py:202]: nice try and except here to ensure no errors are raised when we can't connect to the database.

**Review by ErvinRex — APPROVED** [review]
Great work implementing the word cloud!

**Pull request merged**